benchmark_runner/common/ocp_resources/create_odf.py

```diff
@@ -69,4 +71,7 @@ def create_odf(self):
                     self.wait_for_ocp_resource_create(resource='odf',
                                                       verify_cmd='oc get pod -n openshift-storage | grep osd | grep -v prepare | wc -l',
                                                       count_openshift_storage=True)
+            # sleep between each resource run for avoiding installation failure
+            logger.info(f"sleep {self._environment_variables_dict.get('bulk_sleep_time', '')} seconds")
+            time.sleep(int(self._environment_variables_dict.get('bulk_sleep_time', '')))
```

Comment:
Can you wait on a condition instead?  In code removed above there was a comment explaining why you can't wait on a condition in that case; if that's the situation here, add that comment.

Reply:
I dont know if that the reason that ODF installation failed. 
Try to add this sleep time and see if Installation will pass successfully 
If not I will revert it back !!!

Reply:
My concern is that this will work unreliably because it won't get at the root cause of the problem.